seed/java-sdk/imdb/disable-required-property-builder-checks/src/main/java/com/seed/api/core/SeedApiHttpResponse.java

```diff
@@ -0,0 +1,31 @@
+package com.seed.api.core;
+
+import okhttp3.Response;
+
+public final class SeedApiHttpResponse<T> {
```

Comment:
Naming looks fine to me - alternative is `SeedApiRawResponse`. @dsinghvi any thoughts here for Java?